test/unit/rails_sqli_test.rb

```diff
@@ -8,7 +8,7 @@ def test_action_controller_params_are_escaped
     # No error should be raised on that and the SQL
 
     CMS::Section.find_by(params)
-    sanitized = ActiveRecord::Base.sanitize params['system_name'].to_s
+    sanitized = ActiveRecord::Base.connection.quote params['system_name'].to_s
```

Comment:
I don't know if we should keep it or not. It has to be tested to see if it serves any purpose with current rails or not. Actually it was Hery's suggestion to try delete it. It might be safe to leave it if there is no time to investigate.

Reply:
@hallelujah If  we keep initializer file then we need to apply some workarounds given to fix this issue in particular test cases which are failing as I did in the commit, but If we keep initializer then my first commit is sufficient and fix the issue. Let me know you thoughts so we can fix and close this PR.